Frontend/ui-driver/src/Screens/SubscriptionScreen/View.purs

```diff
@@ -362,7 +386,7 @@ plansBottomView push state =
           ][ textView $
               [ weight 1.0
               , height WRAP_CONTENT
-              , width $ V $ getWidthFromPercent 70
+              , width $ V $ JB.getWidthFromPercent 70
```

Comment:
both weight and width will not work together. remove any one of them based on the requirement